src/scripts/orderBy.ts

```diff
@@ -0,0 +1,19 @@
+/*--------------------------------------------------------------------------
+   Definitions of the supported orderBy parameters.
+ -------------------------------------------------------------------------*/
+
+export enum Parameter {
+	lastModifiedTime = "lastModifiedTime",
+	createdTime = "createdTime",
+	name = "name"
+}
+
+export enum Direction {
+	asc = "asc",
+	desc = "desc"
+}
+
+export class OrderBy {
+	public parameter: Parameter;
+	public direction?: Direction;
```

Comment:
I'm wondering if we should force the dev to specify a direction always to eliminate ambiguity